Index: surf/config.def.h
===================================================================
--- surf.orig/config.def.h
+++ surf/config.def.h
@@ -197,3 +197,5 @@ static Button buttons[] = {
 	{ OnAny,        0,              9,      clicknavigate,  { .i = +1 },    1 },
 	{ OnMedia,      MODKEY,         1,      clickexternplayer, { 0 },       1 },
 };
+
+#define HOMEPAGE "https://duckduckgo.com/"
Index: surf/surf.c
===================================================================
--- surf.orig/surf.c
+++ surf/surf.c
@@ -2140,7 +2140,11 @@ main(int argc, char *argv[])
 	if (argc > 0)
 		arg.v = argv[0];
 	else
+#ifdef HOMEPAGE
+		arg.v = HOMEPAGE;
+#else
 		arg.v = "about:blank";
+#endif
 
 	setup();
 	c = newclient(NULL);
