Index: dwmblocks/blocks.def.h
===================================================================
--- dwmblocks.orig/blocks.def.h
+++ dwmblocks/blocks.def.h
@@ -1,9 +1,14 @@
 //Modify this file to change what commands output to your statusbar, and recompile using the make command.
 static const Block blocks[] = {
 	/*Icon*/	/*Command*/		/*Update Interval*/	/*Update Signal*/
-	{"Mem:", "free -h | awk '/^Mem/ { print $3\"/\"$2 }' | sed s/i//g",	30,		0},
+	{" ",		"/usr/local/sbin/statusbar/cpu.sh",			30,		1},
+	{"  ", 	"/usr/local/sbin/statusbar/mem.sh",			15,		2},
+	{"", 		"/usr/local/sbin/statusbar/temp.sh",			15,		3},
+	{" ",		"/usr/local/sbin/statusbar/vol.sh",			15,		4},
+//	{"Mem:", "free -h | awk '/^Mem/ { print $3\"/\"$2 }' | sed s/i//g",	30,		0},
 
-	{"", "date '+%b %d (%a) %I:%M%p'",					5,		0},
+//	{"", "date '+%b %d (%a) %I:%M%p'",					5,		0},
+	{"", 		"/usr/local/sbin/statusbar/time.sh", 			15, 		5},
 };
 
 //sets delimiter between status commands. NULL character ('\0') means no delimiter.
